src/tools/rbd_mirror/image_sync/SyncPointCreateRequest.cc

```diff
@@ -113,7 +113,8 @@ void SyncPointCreateRequest<I>::send_create_snap() {
     SyncPointCreateRequest<I>, &SyncPointCreateRequest<I>::handle_create_snap>(
       this);
   m_remote_image_ctx->operations->snap_create(
-    cls::rbd::UserSnapshotNamespace(), sync_point.snap_name.c_str(), ctx);
+    cls::rbd::UserSnapshotNamespace(), sync_point.snap_name.c_str(), 0,
```

Comment:
Since this is only used for journaling mirror-mode, we can skip the quiesce request.